src/py42/_internal/clients/storage/archive.py

```diff
@@ -100,7 +100,7 @@ def start_restore(
         show_deleted=None,
         restore_full_path=None,
         timestamp=None,
-        exceptions=None,
+        exclusions=None,
```

Comment:
are these parameters still needed? If they aren't going to be used anymore, i'd rather just remove them from here.

Reply:
All of the unused ones? I removed `exclusions` from the higher level calls. There are many examples in py42 where modules don't use all the request params of client methods.

I will remove `exclusions` here since it is faulty, but it still is technically a request parameter.

If there are more parameters you are referring to (on any client), let me know and I can remove those too.